SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2984,6 +2985,12 @@ void CDiscAdjFEAIteration::SetDependencies(CSolver *****solver, CGeometry ****ge
     break;
   }
 
+  /*--- FSI specific dependencies. ---*/
+  if(fsi) {
+    /*--- Set relation between solution and predicted displacements, which are the transferred ones. ---*/
+    dir_solver->PredictStruct_Displacement(nullptr, config[iZone], solver[iZone][iInst]);
+  }
+
```

Comment:
Since we transfer "predicted displacements", and not directly the solution, there is this extra dependency.
@cvencro this may need some attention for time domain if a higher order predictor is used.